pages/common/exiv2.md

```diff
@@ -0,0 +1,28 @@
+# exiv2
+
+> Image metadata manipulation tool.
+> More information: <https://www.exiv2.org/manpage.html>.
+
+- Print a summary of the image Exif metadata:
+
+`exiv2 {{filename}}`
```

Comment:
Perhaps this will read better? It would be more consistent with other pages.
```suggestion
`exiv2 {{path/to/filename}}`
```

Same goes for the other examples.

Reply:
That seems overly verbose and also technically incorrect. It could be path/to/file but not path/to/filename